bitnami/airflow/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: airflow
-version: 4.3.3
+version: 4.3.4
```

Comment:
You'll need to bump the minor version.
The reason: you're modifying the structure of the **values.yaml**. Therefore, you're affecting users that could be using the previous structure where there was an unique parameter: `resources`